src/includes/getting-started-primer/flutter.mdx

```diff
@@ -26,6 +27,7 @@ Features:
     - [Dio HTTP library](/platforms/dart/configuration/integrations/dio/#performance-monitoring-for-http-requests)
     - [Cold and warm app start](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#app-start-instrumentation)
     - [Slow and frozen frames](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+    - [AssetBundle Instumentation](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#assetbundle-instrumentation)
```

Comment:
```suggestion
    - [AssetBundle Instrumentation](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#assetbundle-instrumentation)
```

Reply:
ups, thanks.

Reply:
It was a copy and paste error, so fixed everywhere.